### PR TITLE
[FIX] mass_mailing: fix mobile preview

### PR DESCRIPTION
**Pull request by pyduf**
The preview of a mailing was displayed twice:
- after a save and when clicking on the preview button
- when double-clicking on the preview button
This solves those problems.

Technical notes: quick explanation of the fixes:
- double display after a save: the click event was registered twice (same for
the code preview button), the fix unregister the click before registering it
again.
- double display when double-clicking quickly on the button: the modal is not
displayed immediately and the button remains active a brief moment allowing a
second click. The fix disables the button when the button is clicked and
re-enables it when the modal is closed.

Task-3047479

Description of the issue/feature this PR addresses:

Current behavior before PR:

Desired behavior after PR is merged:

---
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

File: addons/mass_mailing/static/src/js/mass_mailing_html_field.js [hunk 1]
```diff
@@ -230,15 +232,19 @@ export class MassMailingHtmlField extends HtmlField {
...
+            $previewBtn.prop('disabled', true); // Prevent double execution when double-clicking on the button
             let mailingHtml = new DOMParser().parseFromString(this.wysiwyg.getValue(), 'text/html');
             [...mailingHtml.querySelectorAll('a')].forEach(el => {
                 el.style.setProperty('pointer-events', 'none');
             });
             this.mobilePreview = this.dialog.add(MassMailingMobilePreviewDialog, {
                 title: this.env._t("Mobile Preview"),
                 preview: mailingHtml.body.innerHTML,
+            }, {
+                onClose: () => $previewBtn.prop('disabled', false),
```
Comment (std-odoo) [inline addons/mass_mailing/static/src/js/mass_mailing_html_field.js:247]: can't we use removeAttr instead ? (so the attribute is not in the DOM anymore)

**Review by std-odoo — COMMENTED** [review]
@pyduf Hi :) 

One small comment, otherwise LGTM :)

File: addons/mass_mailing/static/src/js/mass_mailing_html_field.js (hunk 1 above)
Comment (pyduf) [reply addons/mass_mailing/static/src/js/mass_mailing_html_field.js:247]: Good remark but after reading some posts and the following doc, it seems that using prop is the correct way as the attribute represents the initial value while the prop the current value (so modifying the attribute afterwards could not change the property value):
https://jquery.com/upgrade-guide/3.0/#feature-new-signature-for-jquery-get-and-jquery-post
What do you think ?

**Comment by tde-banana-odoo** [comment]
@robodoo r+

LGTM, thanks :)

**Comment by tde-banana-odoo** [comment]
@robodoo r+

**Pull request closed without merging**